### PR TITLE
Update the Upgrading Minikube topic

### PR DESCRIPTION
**Pull request by viniciusbds**
Related to #19861

This PR update the [Upgrading Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/#upgrading-minikube) topic.

### CODE REVIEW

File: content/en/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -332,8 +332,8 @@ Starting the cluster again will restore it to its previous state.
 The `minikube delete` command can be used to delete your cluster.
 This command shuts down and deletes the Minikube Virtual Machine. No data or state is preserved.
 
-### Upgrading minikube
-See [upgrade minikube](https://minikube.sigs.k8s.io/docs/start/macos/)
+### Upgrading Minikube
+If you are using macOS, see [Upgrading Minikube](https://minikube.sigs.k8s.io/docs/start/macos/#upgrading-minikube) to upgrade your existing minikube installation
```
Comment (tengqm) [inline content/en/docs/setup/learning-environment/minikube.md]: Please add a dot to the end of the sentence. Not a big deal, just to trigger netlify build again which is currently failing.

Comment (viniciusbds) [reply content/en/docs/setup/learning-environment/minikube.md]: Done

**Pull request merged**